public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -255,9 +261,13 @@ function AdminGSVLabelView(admin) {
             data: JSON.stringify(data),
             dataType: 'json',
             success: function (result) {
+                button.style.cursor = "pointer";
                 self.commentTextArea.val('');
-            },
+                self.commentButton.popover('toggle');
+                setTimeout(function(){self.commentButton.popover('toggle'); }, 1500);
```

Comment:
more nit-picking! can you add a space after the `{` in all the places where you do this? there are a few in the code